src/applications/gi/components/profile/EstimatedBenefits.jsx

```diff
@@ -46,35 +47,33 @@ const CalculatorResultRow = ({
   screenReaderSpan,
 }) =>
   visible ? (
-    <div
+    <li
       id={`calculator-result-row-${createId(id == null ? label : id)}`}
       className={classNames('row', 'calculator-result', {
         'vads-u-font-weight--bold': bold,
       })}
     >
-      <div className="small-6 columns">
-        {header ? <h4>{label}:</h4> : <div>{label}:</div>}
-      </div>
-      <div className="small-6 columns vads-u-text-align--right">
-        {header ? (
-          <div>
-            <p
-              className="vads-u-font-size--h5 vads-u-font-family--serif
+      {header ? (
```

Comment:
Is it worth splitting this `CalculatorResultRow` function into two smaller portions? The nested if/else logic took a bit to trace. Maybe not, but worth an ask.

Reply:
Probably not. That method is a react component we are using to limit code duplication.